client/my-sites/checkout/checkout-thank-you/transfer-pending.jsx

```diff
@@ -84,9 +125,9 @@ class TransferPending extends PureComponent {
 					title="Checkout Pending"
 					properties={ siteSlug && { site: siteSlug } }
 				/>
+				<WordPressLogo size={ 180 } className="checkout-thank-you__wordpress-logo is-large" />
 				<EmptyContent
-					illustration={ '/calypso/images/illustrations/illustration-shopping-bags.svg' }
-					illustrationWidth={ 500 }
+					illustration={ false }
```

Comment:
`illustration` prop is declared to be a string so an empty string should be used.
As is, following warning appears in the Console:

<img width="1205" alt="screenshot_910" src="https://user-images.githubusercontent.com/127594/64360911-4fbd7e80-cfc0-11e9-85a6-3130b8ec3589.png">